src/Shared/PackageManagers/NPMProjectManager.cs

```diff
@@ -190,7 +203,7 @@ public override async Task<IEnumerable<string>> EnumerateVersionsAsync(PackageUR
 
         public override Uri GetPackageAbsoluteUri(PackageURL purl)
```

Comment:
What's the difference between this Url and artifact download url?

Reply:
Found this in the [`BaseProjectManager`](https://github.com/microsoft/OSSGadget/blob/main/src/Shared/PackageManagers/BaseProjectManager.cs#L350). Maybe the method name can be changed later on as part of other enhancements.

Reply:
This brings you to, for example, https://registry.npmjs.org/lodash, vs https://registry.npmjs.org/lodash/-/lodash-4.17.15.tgz for the artifactUri. It doesn't link to a download uri, and it doesn't specify a version.